Automattic/wp-calypso
# Jetpack: update Jetpack Search illustration asset

#### Changes proposed in this Pull Request

* Removes old style illustration.
* Introduces a new illustration asset.

#### Testing instructions

* Test in WordPress.com
  1. Go to `http://calypso.localhost:3000/jetpack-search/SITE_URL` where `SITE_URL` is a Jetpack site with and without a Search subscription.
  1. Go to `http://calypso.localhost:3000/jetpack-search/SITE_URL` where `SITE_URL` is a WordPress.com simple site with and without a Search subscription.
* Test in Jetpack.com
  1. Go to `http://jetpack.cloud.localhost:3001/jetpack-search/SITE_URL` where `SITE_URL` is a Jetpack site (currently connected) with and without a Search subscription.
* Make sure you see the new illustration in all scenarios.

Related to 1199980337313591-as-1200819373070128


#### Screenshots

Before | After
--|--
![image](https://user-images.githubusercontent.com/390760/131852268-a3b53e7f-88cc-45d5-9eb0-c00e85bbdca4.png) | ![image](https://user-images.githubusercontent.com/390760/132020454-2bb0c3ee-5c40-47f8-ac88-b26c06e2b02c.png)
![image](https://user-images.githubusercontent.com/390760/131852334-cd89b16d-3d87-4d21-ad2b-f2a9b4d62a3a.png) | ![image](https://user-images.githubusercontent.com/390760/132020372-08f401e4-96d4-4832-82f9-2ac6d17f4059.png)
